src/cr/cube/crunch_cube.py

```diff
@@ -713,6 +714,36 @@ def zscore(self, weighted=True, prune=False, hs_dims=None):
         res = [s.zscore(weighted, prune, hs_dims) for s in self.slices]
         return np.array(res) if self.ndim == 3 else res[0]
 
+    def pairwise_chisq(self, axis=0):
+        """Return square ndarray of pairwise Chi-square along axis.
+
+        Zscore is a measure of statistical signifficance of observed vs.
+        expected counts. It's only applicable to a 2D contingency tables.
+        For 3D cubes, the measures of separate slices are stacked together
+        and returned as the result.
+
+        :param weighted: Use weighted counts for zscores
+        :param prune: Prune based on unweighted counts
+        :param hs_dims: Include headers and subtotals (as NaN values)
+        :returns zscore: ndarray representing zscore measurements
+        """
+        res = [s.pairwise_chisq(axis=0) for s in self.slices]
+        return res[0]
```

Comment:
I'm not sure that this is right. I'd expect us to stack the results of slices into a list or something... that is, I'd expect `return res` to be the correct value. And then it's the job of the exporter to discern between tabs, if there are multiples thereof...

Reply:
yeah this was just the expedient 'give me some numbers' assuming only one slice.